src/app/infinitescroll/infinitescroll.directive.ts

```diff
@@ -13,7 +13,8 @@ import {
     selector: '[almInfiniteScroll]',
     exportAs: 'almInfiniteScroll'
 })
-export class InfiniteScrollDirective implements OnInit {
+
```

Comment:
Why did you add a line between the class and the annotation? This may be minor but I just don't understand why.

Reply:
Just to make it more readable.

Reply:
this is not more readable as it violates the style standard